test/cases/expressions/xhp-class-name.hack

```diff
@@ -0,0 +1 @@
+assert_func(:page:subpage:test::class);
```

Comment:
Nice! Good to have this regression covered. Didn't ever think about this being an issue but it makes sense given the use of colon in the two cases.

One nit: Hacklang uses "classname" all one word to refer to [class type instances](https://docs.hhvm.com/hack/built-in-types/classname) (I actually didn't know before looking it up). At first I was confused thinking this test was meant for the actual naming of a class — e.g. `class Name { ... }`. Let's change this to `xhp-classname.hack`

Reply:
Haha I actually named this `classname` at first but then didn't remember why when committing so added the dash. Flipped back now.